gatovid/api/game.py

```diff
@@ -1,13 +1,57 @@
 """
 """
 
-from flask import Blueprint
+from flask import session
```

Comment:
y por que lo importas de aqui y no de las extensiones?? no entiendo

Reply:
yo tampoco lo entiendo, al parecer las extensiones solo añaden el manager de sesiones, pero no el `session`.

Si miras en la documentación (https://flask-session.readthedocs.io/en/latest/), dice lo siguiente:
`The Session instance is not used for direct access, you should always use flask.session`

Reply:
Lol pues nada un poco raro pero me vale